src/messages.ts

```diff
@@ -4,40 +4,51 @@ const cashbackIBAN1 = {
     '---\nit:\n    cta_1: \n        text: "Inserisci IBAN"\n        action: "ioit://CTA_BPD_IBAN_EDIT"\nen:\n    cta_1: \n        text: "Add IBAN"\n        action: "ioit://CTA_BPD_IBAN_EDIT"\n---\n\n\nL\'Extra Cashback di Natale è finito il 31 Dicembre!\n\nSembra che tu **non abbia ancora inserito un IBAN** valido per l\'accredito dell\'eventuale rimborso sul tuo conto.\n\n**Inserisci il tuo IBAN entro il 9 gennaio 2021** per essere sicuro di ricevere il relativo bonifico.\n\nIl calcolo definitivo delle transazioni e dei rimborsi si concluderà l\'11 gennaio 2021, per avere il tempo di raccogliere tutte le transazioni valide.\n\nTi ricordiamo che solo se hai totalizzato un numero di almeno 10 transazioni valide, riceverai un rimborso pari all\'importo mostrato in app, direttamente sull\'IBAN che ci hai comunicato, fino a un massimo di 150 euro.\n\nIl Cashback continua: sarai automaticamente iscritto al nuovo periodo 1 gennaio - 30 giugno 2021!\n\nGrazie per aver partecipato all\'Extra Cashaback di Natale!\n\n',
 };
 
+const cashbackTransaction1 = {
+  subject: 'Importante: verifica le tue transazioni per il Cashback',
+  markdown:
+    'Ciao! Stiamo effettuando delle verifiche su alcune **transazioni anomale** rilevate nell’ambito del Cashback: si tratta di transazioni **ricorrenti di importo irrisorio**, effettuate in numero elevato **presso lo stesso esercente, lo stesso giorno** e che, pertanto, appaiono **non qualificabili come "acquisti"** di beni o servizi, ai sensi del Programma.\n\nQueste transazioni, se legate a condotte abusive, non darebbero diritto al alcun tipo di rimborso. Per questo, abbiamo provveduto a **stornare le operazioni “sospette”** dal sistema: puoi visualizzarle con il segno "–" nel dettaglio della tua card del Cashback.\n\n**Se pensi si tratti di un errore e le tue transazioni sono in regola, hai 7 giorni dalla data di invio di questo messaggio per provare che ci stiamo sbagliando!** Usa [questo modulo online](https://forms.gle/mHwWGUeCpb3WY6qK9) per dichiarare l’oggetto effettivo dell’acquisto a cui corrisponde ciascuna delle transazioni stornate.\n\nRicevuto il tuo modulo, le tue transazioni - **se ritenute valide** ai fini del Cashback - **torneranno a essere conteggiate** ai fini del rimborso. Al contrario, saranno eliminate definitivamente dal Programma; questo accadrà anche in mancanza di un tuo riscontro a questa comunicazione entro i 7 giorni dall’invio.\n\n',
```

Comment:
@pasqualedevita there's a typo—better to fix in case we'll need this template again:

[...]non darebbero diritto ~al~ **ad** alcun tipo di rimborso.[...]

Could you please fix it? 🙏